src/wp-includes/functions.php

```diff
@@ -7626,7 +7626,7 @@ function get_dirsize( $directory, $max_execution_time = null ) {
  */
 function recurse_dirsize( $directory, $exclude = null, $max_execution_time = null, &$directory_cache = null ) {
 	$directory  = untrailingslashit( $directory );
-	$cache_path = untrailingslashit( str_replace( ABSPATH, '', $directory ) );
+	$cache_path = untrailingslashit( str_replace( WP_CONTENT_DIR, '', $directory ) );
```

Comment:
I think this needs to be duplicated in `clean_dirsize_cache()` -- a little unclear on the purpose of the change but will check this out myself.

Reply:
yeah, i haven't gotten that far yet, but it's on my list. just changing the const isn't enough, but i think if the mock function is updated that should do it.

if you have time to look into it, that'd be very much appreciated, but not worries if not